app/views/proposals/_archive_link.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="col-sm-2">
-  <%- if @approved_data.rows.size > @CLOSED_PROPOSAL_LIMIT %>
+  <%- unless @approved_data.rows.size < @CLOSED_PROPOSAL_LIMIT %>
```

Comment:
Mind adding a test for this? Also, would be great to have this as a separate PR from the `FORCE_USER_ID` thing.